stubs/_pytest/mark/__init__.pyi

```diff
@@ -4,3 +4,4 @@ from .structures import (
     MarkGenerator as MarkGenerator,
     get_empty_parameterset_mark as get_empty_parameterset_mark,
 )
+from .structures import MARK_GEN
```

Comment:
cc @phil-lopreiato getting this error when running pyre
<img width="853" alt="Screen Shot 2020-05-30 at 1 06 45 PM" src="https://user-images.githubusercontent.com/516458/83334862-5fc86d00-a277-11ea-879d-f8c1ef72c4a7.png">

Weirdly when I do a stubgen I get wildly different generated files then what's in the repo. Are we using different versions of things?

```
$ pytest --version
This is pytest version 5.4.2, imported from /Users/zach/Desktop/the-blue-alliance/venv/lib/python3.7/site-packages/pytest/__init__.py
setuptools registered plugins:
  pytest-cov-2.9.0 at /Users/zach/Desktop/the-blue-alliance/venv/lib/python3.7/site-packages/pytest_cov/plugin.py
$ mypy --version
mypy 0.770
```

Reply:
seems like this is resolved in the latest push?

Reply:
It's not :\